### PR TITLE
Smaller quickstart tiles

### PR DESCRIPTION
**Pull request by aswanson-nr**
## Description

Decreases the size of the quickstart tiles.

### CODE REVIEW

**Comment by aswanson-nr** [comment]
If you see any issues with tile images, try pulling in the latest quickstarts by running `yarn fetch-quickstarts`. The newer images should work without issue.

**Comment by aswanson-nr** [comment]
Should be merged after https://github.com/newrelic/developer-website/pull/1984

**Comment by josephgregoryii** [comment]
Checking the build on this, on the list view for I/O it seems that the images do not load. Works fine for grid view

<details>
<summary>see here</summary>

Screen Shot 2022-01-03 at 2 13 32 PM


</details

**Comment by aswanson-nr** [comment]
The latest commit should fix the styling issues on the list view, @josephgregoryii !

**Comment by aswanson-nr** [comment]
Still having some issues with images being very small on certain breakpoints

**Review by clarkmcadoo — APPROVED** [review]

**Comment by aswanson-nr** [comment]
Should be ready for full review now!

File: src/components/PackImg.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ const PackImg = ({ className, logoUrl, packName }) => {
       <img
         css={css`
           display: block;
+          max-width: 100%;
+          max-height: 100%;
         `}
         src={logoUrl}
         alt={packName}
```
Comment (aswanson-nr) [inline src/components/PackImg.js:31]: Lower the pack acronym text size

File: src/components/PackTile.js [hunk 2]
```diff
@@ -73,95 +73,122 @@ const PackTile = ({
...
+        --tile-image-height: 100px;
+        padding: 1rem;
         overflow: hidden;
-        display: flex;
-        flex-direction: column;
-
-        ${view === VIEWS.LIST &&
+        display: grid;
+        grid-gap: 0.2rem;
+        grid-template-rows: var(--tile-image-height) 3rem 1fr 0.2fr;
```
Comment (zstix) [inline src/components/PackTile.js:81]: Perhaps we want to add a comment to what each of these row heights represent (image, title, desc, chip).

File: src/components/PackTile.js [hunk 3]
```diff
@@ -73,95 +73,122 @@ const PackTile = ({
...
 
-        @media screen and (max-width: ${QUICKSTARTS_COLLAPSE_BREAKPOINT}) {
-          flex-direction: row;
-        }
+        ${isListView() &&
+        css`
+          grid-template-columns: 0.5fr 1fr 1fr;
+          grid-template-areas:
+            'image title tag'
+            'image summary tag';
```
Comment (zstix) [inline src/components/PackTile.js:95]: As we discussed on the campfire, we might be able to get away with simplifying the styles by using grid for both the grid layout and list layout: https://codepen.io/zstix/pen/WNZKrKe?editors=1100

File: src/pages/instant-observability.js [hunk 4]
```diff
@@ -498,28 +499,35 @@ const QuickstartsPage = ({ data, location }) => {
...
             css={css`
               display: grid;
-              grid-gap: 1rem;
+              grid-gap: 1.25rem;
               grid-template-columns: repeat(4, 1fr);
-              grid-auto-rows: minmax(var(--guide-list-row-height, 150px), auto);
+              grid-auto-rows: minmax(
+                var(--guide-list-row-height, 150px),
+                287px
+              );
```
Comment (zstix) [inline src/pages/instant-observability.js:507]: We might want to double-check this is working as intended. After discussing it, I don't think we want these to have variable heights.

**Review by zstix — COMMENTED** [review]
We have some possible cleanup opportunities, but this looks great!

File: src/components/PackTile.js [hunk 5]
```diff
@@ -73,95 +73,134 @@ const PackTile = ({
       className={className}
       interactive
       css={css`
+        --tile-image-height: 100px; /* Logo image height */
```
Comment (clarkmcadoo) [inline src/components/PackTile.js:76]: Was having issues with this, seeing the icon pop outside of the white background, was able to fix it by changing this to `100%`

Screen Shot 2022-01-06 at 2 45 27 PM

Comment (clarkmcadoo) [reply src/components/PackTile.js:76]: This is on grid view, desktop

Comment (aswanson-nr) [reply src/components/PackTile.js:76]: This should be fixed with the latest commit!

**Review by clarkmcadoo — APPROVED** [review]

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**